bioimageio_collection_backoffice/s3_client.py

```diff
@@ -223,6 +223,9 @@ def _cp_dir(self, src: str, tgt: str):
 
         return objects
 
+    def rm(self, object: str):
+        self._client.remove_object(self.bucket, object)
```

Comment:
We should add the prefix here

Reply:
all good with the way I added the prefix @FynnBe?